flujo/application/core/context_vars.py

```
@@ -1,12 +1,15 @@
+"""Compatibility shim for `context_vars` (moved to core/context/)."""
 
 from __future__ import annotations
-from contextvars import ContextVar
-from typing import Optional
 
-# Cache enable override to avoid mutating shared core state (used by loop/state-machine policies).
-_CACHE_OVERRIDE: ContextVar[Optional[bool]] = ContextVar("CACHE_OVERRIDE", default=None)
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from flujo.application.core.context.context_vars import *  # noqa: F401,F403
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)